pkgs/applications/blockchains/bitcoin.nix

```diff
@@ -14,6 +18,16 @@ stdenv.mkDerivation rec{
     sha256 = "5c7d93f15579e37aa2d1dc79e8f5ac675f59045fceddf604ae0f1550eb03bf96";
   };
 
+  desktop = optional withGui (fetchurl {
+    url = "https://raw.githubusercontent.com/bitcoin-core/packaging/${majorVersion}/debian/bitcoin-qt.desktop";
```

Comment:
I'd prefer you use [`majorMinor`](https://github.com/NixOS/nixpkgs/blob/909904d0ae8750cd6df237b7b439fd2ffeba6273/lib/versions.nix#L45) instead of defining your own `majorVersion`.

Reply:
Thx, implemented